notification/build.gradle

```diff
@@ -18,6 +18,7 @@ apply plugin: 'java'
 apply plugin: 'jacoco'
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
+apply plugin: 'elasticsearch.testclusters'
```

Comment:
do we need this to apply here?

Reply:
good point, we dont need that. Will remove.